pages/create.js

```diff
@@ -0,0 +1,79 @@
+import Link from "next/link.js";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import { Form } from "@/components/form/Form.js";
+import styled from "styled-components";
+
+const StyledContent = styled.div`
+  padding-top: 1.5cm; /* Abstand zum Header */
+  padding-bottom: 1.5cm; /* Ändere die Höhe nach Bedarf, um Platz für den Footer zu schaffen */
+`;
+
+const CreateContainer = styled.div`
+  padding: 20px;
+  background-color: #1e2124;
+  color: #61dafb;
+  position: relative;
+  margin-left: 0.5cm; // Hinzugefügt: Linker Rand
+  margin-right: 0.5cm; // Hinzugefügt: Rechter Rand
+`;
+
+const CreateTitle = styled.h2`
+  font-size: 24px;
+  margin-bottom: 20px; // Hinzugefügt
+`;
+
+const StyledLink = styled.a`
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  padding: 10px;
+  font-size: 18px;
+  background-color: #61dafb;
+  color: #282c34;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+  text-decoration: none;
+
+  &:hover {
+    background-color: #38a169;
+    text-decoration: underline;
+  }
+`;
```

Comment:
```
const StyledContent = styled.div`
  padding-top: 1.5cm; /* Abstand zum Header */
  padding-bottom: 1.5cm; /* Ändere die Höhe nach Bedarf, um Platz für den Footer zu schaffen */
`;

const CreateContainer = styled.div`
  padding: 20px;
  background-color: #1e2124;
  color: #61dafb;
  position: relative;
  margin-left: 0.5cm; // Hinzugefügt: Linker Rand
  margin-right: 0.5cm; // Hinzugefügt: Rechter Rand
`;

const CreateTitle = styled.h2`
  font-size: 24px;
  margin-bottom: 20px; // Hinzugefügt
`;

const StyledLink = styled.a`
  position: absolute;
  top: 20px;
  right: 20px;
  padding: 10px;
  font-size: 18px;
  background-color: #61dafb;
  color: #282c34;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  transition: background-color 0.3s;
  text-decoration: none;

  &:hover {
    background-color: #38a169;
    text-decoration: underline;
  }
`;
```

Same here as mentioned above for your styled components. I think these components are (almost?) the same as for your `edit` page. You could think of putting them then in separate file to reduce the amount of code that is not needed. [And you can work with props if you need to adjust just some declarations inside of your styled components.](https://github.com/neuefische/cgn-web-23-3/blob/main/sessions/react-styled-components-2/react-styled-components-2.md)

Reply:
Thank you for the input. Sometimes the values differ slightly from each other. I would therefore like to do the styling in the respective files (as long as this is possible) so that I have a complete overview and can make changes if necessary.

Reply:
You can do so. 👍  
If you have some time left you can also later think about a good way to sole this for less repetitive code. :)